integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+   * Assigns unique namespaces for operator and domains.
+   * Pull FMW image and Oracle DB image if running tests in Kind cluster. Installs operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(4) List<String> namespaces) {
+
+    logger = getLogger();
+    logger.info("Assign a unique namespace for DB and RCU");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    dbNamespace = namespaces.get(0);
+
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    opNamespace = namespaces.get(1);
+
+    logger.info("Assign a unique namespace for FMW domain");
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    fmwDomainNamespace = namespaces.get(2);
+
+    logger.info("Assign a unique namespace for WLS domain");
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    wlsDomainNamespace = namespaces.get(3);
+
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(fmwDomainNamespace);
+    createSecretForBaseImages(wlsDomainNamespace);
+
+    //install Oracle Database Operator
+    assertDoesNotThrow(() -> DbUtils.installDBOperator(dbNamespace), "Failed to install database operator");
+
+    logger.info("Create Oracle DB in namespace: {0} ", dbNamespace);
+    dbUrl = assertDoesNotThrow(() -> DbUtils.createOracleDBUsingOperator(dbName, RCUSYSPASSWORD,
+        dbNamespace, WORK_DIR + "/oracledatabase"));
+
+    logger.info("Create RCU schema with fmwImage: {0}, rcuSchemaPrefix: {1}, dbUrl: {2}, "
+        + " dbNamespace: {3}:", FMWINFRA_IMAGE_TO_USE_IN_SPEC, RCUSCHEMAPREFIX, dbUrl, dbNamespace);
+    assertDoesNotThrow(() -> createRcuSchema(FMWINFRA_IMAGE_TO_USE_IN_SPEC, RCUSCHEMAPREFIX,
+        dbUrl, dbNamespace));
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, fmwDomainNamespace, wlsDomainNamespace);
+  }
+
+  /**
+   * Create a basic FMW model in image domain. Create the FMW domain with introspectVersion After domain is created and
+   * introspector exists restart the operator. Verify the restarted operator can find the existing introspector and wait
+   * for it to complete rather than replacing a new introspector. Verify Pod is ready and service exists for both admin
+   * server and managed servers. Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Create FMW Domain model in image")
+  void  testFmwModelInImageWithDbOperator() {
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createOcirRepoSecret(fmwDomainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(fmwAminSecretName,
+        fmwDomainNamespace,
+        "weblogic",
```

Comment:
You can use the constants defined in TestConstants.java for default username and password.

Reply:
done